src/atomic_mass_data.cpp

```diff
@@ -0,0 +1,83 @@
+#include "openmc/atomic_mass_data.h"
+
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+
+// Clean string to remove unrequired spaces
+std::string remove_space(std::string s)
+{
+  s.erase(std::remove_if(s.begin(), s.end(), ::isspace), s.end());
+  return s;
+}
+
+// Read a line from atomic mass data and pull usesull content
+atomic_data read_atomic_data(std::string line)
+{
+  atomic_data atom_data;
+  atom_data.mass_excess =
+    std::stod(line.substr(29, 6) + "." + line.substr(36, 6));
+  atom_data.binding_energy =
+    std::stod(line.substr(55, 6) + "." + line.substr(62, 6));
+  atom_data.mass =
+    std::stod(line.substr(106, 3)) +
+    1e-6 * std::stod(line.substr(110, 6) + '.' + line.substr(117, 3));
+  return atom_data;
+}
+
+namespace openmc {
+namespace data {
+
+AtomicData::AtomicData(std::string data_file)
+{
+  // Open Data file
+  std::ifstream myfile(data_file);
+  if (!myfile) {
+    std::cout << "NO file found in " << data_file << std::endl;
+  }
+  // Skip header
+  for (int i = 0; i < line_in_header; i++) {
+    std::string line;
```

Comment:
Maybe you could move `std::string line` above this loop so that it doesn't need to be defined again below?